client/src/reviewService.jsx

```diff
@@ -19,24 +19,26 @@ class ReviewService extends React.Component {
     this.setReviewsFilteredBySearch = this.setReviewsFilteredBySearch.bind(this);
     this.setReviewsFilteredByTier = this.setReviewsFilteredByTier.bind(this);
     this.setReviewsFilteredBySearchAndTier = this.setReviewsFilteredBySearchAndTier.bind(this);
-
+    this.showTwelveMoreReviews = this.showTwelveMoreReviews.bind(this);
     this.state = {
-      courseId: null,
+      courseId: Number(querystring.parse(window.location.search)['?courseId']),
```

Comment:
We've just been talking about this, of course. I'll need to figure out some hooks equivalent of this.